functions/assignOrder/source.js

```diff
@@ -0,0 +1,23 @@
+
+/*
+This function takes in an order id and a driver id and updates the assignedToDriver field in the order.
+// how to test on the stitch ui:
+// once you have gone to the function ahad_test, click "console" and paste the commented out code below. Then, copy and paste the real function in the function editor and click run.
+exports('5e76aad15f06170a32d8f8b5', '2')
+
+*/
+
+exports = function(orderId, driverId){
+
+  let db = context.services.get("mongodb-atlas").db("stayneighbor")
+  let collection = db.collection("orders")
+  let query = {_id: BSON.ObjectId(orderId)}
+  let updateCmd = {$set: {assignedToDriver: driverId}}
```

Comment:
can we also set the assignedToOrgId, you will be able to get it from:

context.user.custom_data.organizationId

Reply:
Good idea. I made the change but am not really sure how to test it. I'd need to create a user on the ui right?